src/runtime-rs/crates/hypervisor/src/dragonball/vmm_instance.rs

```diff
@@ -152,6 +154,7 @@ impl VmmInstance {
     }
 
     pub fn put_boot_source(&self, boot_source_cfg: BootSourceConfig) -> Result<()> {
+        METRICS.read().unwrap().request.configure_boot_source.inc();
```

Comment:
should we add metrics here or add it to vmm_action.rs?

Reply:
Maybe not as useful. I removed it.

Reply:
You misunderstood my words.
I mean you can move the code in your patch to dragonball's vmm_action.rs, don't put it here in runtime's code

Reply:
Okay, I moved the code to  vmm_action.rs except for the request using `handle_request_with_retry`.